# Exception counter

Write a function `count_exceptions(f, n1, n2)` that receives a function `f` and two integers `n1` and `n2`. Function `f` accepts an integer and may throw an exception when applied to an invalid integer. The given function is applied to all integers in the range `[n1, n2]` and returns the amount of exceptions caught.



## Private Tests [100%]

## Public Tests [100%]
